### ISSUE TITLE
Support installation via poetry

### ISSUE THREAD

**Issue by OpenBagTwo** [OpenBagTwo/EnderChest#16]
Make any modifications necessary to ensure that this package can be installed via `poetry`, and then add `poetry` installation instructions to the README and install docs (if done after #12)

Out of scope: building and deploying using poetry

**Comment by OpenBagTwo** [comment]
I *believe* I get this for free once the package is on PyPI...

**Comment by OpenBagTwo** [comment]
Yeah, it's installable via poetry. But I guess that only matters if someone wants to use it as a dependency?
